src/components/aeIdentityMain/aeIdentityMain.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div @click="forwardEvent" :class="classObject">
+    <div class="flex-row">
+      <ae-identity-avatar
+        class="ae-identity__avatar"
+        :address='identity ? identity.address : ""'
+        @click="forwardEvent"
+      />
+      <span :class="['identity-name-position', collapsedModifier]">
+        <header :class="['identity-name', collapsedModifier]">{{identityName}}</header>
+        <small class="truncated-address" v-if="collapsed" data-truncated-address>{{shortAddress}}  ••••••</small>
+      </span>
+      <div class="balances">
+        <div v-if='tokenAmount' class="balance _bold">
+          <span class="amount" data-token-amount>{{tokenAmount}}</span>
+          <span class="currency-symbol">AE</span>
+        </div>
+        <div class="balance">
+          <span class="amount" data-amount>{{amount}}</span>
```

Comment:
`data-amount`, `data-token-amount`, `data-truncated-address` it is just for tests? You shouldn't write here anything for tests.

Reply:
Well you need to select these elements and check their values somehow. If I use the class names as selectors than changing them will break the tests unnecessarily.   Adding something for better testability is better than not  having tests, right?

Reply:
You should use class names instead. It is okay to update tests if you make some changes to the corresponding component.